api/src/app.py

```diff
@@ -22,6 +22,7 @@
 
 if ENVIRONMENT == 'development':
     metrics = PrometheusMetrics(app)
+    app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///:memory:'
```

Comment:
Some value needs to be set for `production` as well.

Reply:
will include a provisional solution but left it out intentionally since i am not sure about the final location for the db. also, i think it would be beneficial if we have a development db on disc, too. what do you think?

Reply:
provided